net.solarnetwork.node.weather.ibm.wc/OSGI-INF/blueprint/module.xml

```diff
@@ -0,0 +1,261 @@
+		<osgix:interfaces>
+			<beans:value>net.solarnetwork.node.job.ManagedTriggerAndJobDetail</beans:value>
+			<beans:value>net.solarnetwork.node.job.ServiceProvider</beans:value>
+			<beans:value>net.solarnetwork.node.settings.SettingSpecifierProvider</beans:value>
+		</osgix:interfaces>
+		<osgix:service-properties>
+			<beans:entry key="settingPid" value="net.solarnetwork.node.weather.ibm.wc.day"/>
+		</osgix:service-properties>
+		<bean class="net.solarnetwork.node.job.SimpleManagedTriggerAndJobDetail">
+
+			<!-- the trigger defines when the periodic job runs; in this case
+			     we define a cron style trigger, that by default runs once/minute -->
+			<property name="trigger">
+				<bean class="net.solarnetwork.node.job.RandomizedCronTriggerFactoryBean">
+					<property name="name" value="WCDayDatumLoggerTrigger"/>
+					<property name="cronExpression" value="5 * * * * ?"/>
+					<property name="misfireInstructionName" value="MISFIRE_INSTRUCTION_DO_NOTHING"/>
+					<property name="randomSecond" value="true"/>
+				</bean>
+			</property>
+
+			<!-- we can also publish the FoobarDatumDataSource instance itself as a service
+			     by configuring serviceProviderConfigurations here... this is optional but
+			     generally a good idea -->
+			<property name="serviceProviderConfigurations">
+				<map>
+					<entry key="datumDataSource">
+						<bean class="net.solarnetwork.node.job.SimpleServiceProviderConfiguration">
+							<property name="interfaces">
+								<list>
+									<value>net.solarnetwork.node.DatumDataSource</value>
+								</list>
+							</property>
+							<property name="properties">
+								<map>
+									<entry key="datumClassName" value="net.solarnetwork.node.domain.GeneralDayDatum"/>
+								</map>
+							</property>
+						</bean>
+					</entry>
+				</map>
+			</property>
+
+			<!-- the jobDetail defines what job should be executed periodically by the
+			     trigger; here we define a  net.solarnetwork.node.job.DatumDataSourceManagedLoggerJob
+			     job which will invoke the readCurrentDatum() method on a DatumDataSource, which
+			     is what FoobarDatumDataSource is! -->
+			<property name="jobDetail">
+				<bean class="org.springframework.scheduling.quartz.JobDetailFactoryBean">
+					<property name="name" value="WCDayDatumLoggerJob"/>
+					<property name="jobClass" value="net.solarnetwork.node.job.DatumDataSourceManagedLoggerJob"/>
+					<property name="jobDataAsMap">
+						<map>
+							<entry key="datumDao" value-ref="generalLocationDatumDao"/>
+							<entry key="multiDatumDataSource">
+								<bean class="net.solarnetwork.node.support.LocationDatumDataSource">
+									<property name="messageSource" ref="dayJobMessageSource"/>
+									<property name="delegate">
+										<bean class="net.solarnetwork.node.weather.ibm.wc.WCDayDatumDataSource">
+											<property name="objectMapper" ref="objectMapper"/>
```

Comment:
Instead of injecting an `ObjectMapper`, a `BasicWCClient` should be injected, that refers to a single top-level `<bean>` e.g.

```xml
<property name="client" ref="basicClient"/>
```